frontend/app/routes/$lang+/_public+/apply+/$id+/adult/confirmation.tsx

```diff
@@ -334,25 +339,52 @@ export default function ApplyFlowConfirm() {
         </div>
       </div>
 
-      <Button
-        className="mt-5 print:hidden"
-        size="lg"
-        variant="primary"
-        onClick={(event) => {
-          event.preventDefault();
-          window.print();
-        }}
-        data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form:Print a copy of your application bottom click"
-      >
-        {t('confirm.print-btn')}
-      </Button>
-
-      <fetcher.Form method="post" noValidate className="mt-5 flex flex-wrap items-center gap-3">
-        <input type="hidden" name="_csrf" value={csrfToken} />
-        <Button variant="primary" onClick={() => sessionStorage.removeItem('flow.state')} size="lg" className="print:hidden" data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form:Exit - Confirmation click">
-          {t('apply-adult:confirm.exit')}
-        </Button>
-      </fetcher.Form>
+      <div className="max-w-min">
+        <div className="flex flex-col items-center">
```

Comment:
That `div` should only wrap the pring button with `className="my-6"` and remove the margin on the button.

Reply:
Removed flex and added margin to div